.github/workflows/build_publish.yml

```diff
@@ -13,12 +13,18 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
+          server-id: nexus-sonatype
+          server-username: NEXUS_USERNAME
+          server-password: NEXUS_PASSWORD
       - name: Set AWS credentials
         uses: aws-actions/configure-aws-credentials@v1
         with:
```

Comment:
This `Set AWS credentials` step can be removed as we are not going to deploy to S3.

Reply:
I have removed the POM profile for S3 and also removed the action for S3 for snapshot